### PR TITLE
M0090IDE-4009 Add chapter about useing libraries in Applications

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**

### CODE REVIEW

File: ApplicationDeveloperGuide/sandboxedAppLibraries.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    <dependencies>
+        <dependency org="ej.api" name="bon" rev="1.4.0" />
+    </dependencies>
+
+Add a library in META-INF/libraries folder
+------------------------------------------
+
+When the library used by the Application is not available in a module repository, 
+its JAR file can be directly added in the ``META-INF/libraries`` folder of the Application project.
```
Comment (frivieremicroej) [inline ApplicationDeveloperGuide/sandboxedAppLibraries.rst:27]:
```suggestion
Use an External Library
=======================

A Sandboxed Application can embed external libraries in addition to those declared as :ref:`module dependencies <module-dependencies>`. 

For that, the JAR file must be dropped to the ``META-INF/libraries`` directoy of the Sandboxed Application project.
```

Comment (frivieremicroej) [reply ApplicationDeveloperGuide/sandboxedAppLibraries.rst:27]: @tdelhomenie-microej remind also to add the procedure to add it to eclipse project classpath

**Review by frivieremicroej — CHANGES_REQUESTED** [review]

File: ApplicationDeveloperGuide/sandboxedAppLibraries.rst (hunk 1 above)
Comment (tdelhomenie-microej) [reply ApplicationDeveloperGuide/sandboxedAppLibraries.rst:27]: Your suggestion removes the chapter `Add a library as a dependency`, is it a mistake?

Comment (tdelhomenie-microej) [reply ApplicationDeveloperGuide/sandboxedAppLibraries.rst:27]: > @tdelhomenie-microej remind also to add the procedure to add it to eclipse project classpath

The jars are actually automatically added to the Eclipse project classpath. I guess this is due to the nature `com.is2t.microej.wadapps.workbench.wadappsApplicationNature` that is still present when creating a Sandboxed Application project:
```
	<natures>
		<nature>org.eclipse.jdt.core.javanature</nature>
		<nature>org.apache.ivyde.eclipse.ivynature</nature>
		<nature>com.is2t.microej.wadapps.workbench.wadappsApplicationNature</nature>
	</natures>
```
We should probably adapt this nature to keep only features related to KF-only apps.

File: ApplicationDeveloperGuide/sandboxedAppLibraries.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.. _chapter.application.useLibrary:
+
+Use a Library
```
[the remark this answers is not in the document]
Comment (tdelhomenie-microej) [reply ApplicationDeveloperGuide/sandboxedAppLibraries.rst:3]: Done.

**Review by frivieremicroej — CHANGES_REQUESTED** [review]

**Review by frivieremicroej — APPROVED** [review]

**Pull request merged**